package.json

```diff
@@ -52,13 +52,14 @@
   },
   "dependencies": {
     "@angular/cdk": "5.2.4",
-    "@types/dragula": "2.1.29",
     "@bullhorn/dragula": "1.0.1",
+    "@types/dragula": "2.1.29",
     "ace-builds": "1.3.1",
     "angular2-text-mask": "8.0.4",
     "brace": "0.11.1",
     "date-fns": "1.29.0",
     "hint.css": "2.5.0",
+    "moment": "^2.22.0",
```

Comment:
We do not want moment as a dependency in NovoElements. It is too bloated. We had everything use moment in that past and updated to use `date-fns`.

Reply:
We should find a solution using date-fns without another dep.

Reply:
https://date-fns.org/docs/format

Can still have custom formatting, if pass custom format, disable the mask

Reply:
Good call, I got tunnel vision. I will correct it.